labgrid/external/remote_tmpdir.py

```diff
@@ -0,0 +1,53 @@
+"""
+    RemoteTmpdir is a class that maintains a temporary directory on a
+    remote linux system.
+    Ex. when copying a test script or binary to DUT, RemoteTmpdir assists
+    to prevent pollution of the target.
+
+    RemoteTmpdir is to be created in a fixture, which can take care of a
+    testmodule in a subdirectory. Recommended scope is function, to avoid
+    polution between tests.
+
+    See examples/remote_tmpdir
+"""
+
+import os
+
+
+class RemoteTmpdir:
+
+    # scriptdir is a directory relative to pytest root
+    def __init__(self, shell, testdir=None):
+        stdout, stderr, returncode = shell.run("mktemp -d")
+        assert returncode == 0
+        assert len(stdout) == 1
```

Comment:
or simpler
```python
stdout = shell.run_check("mktemp -d")
assert len(stdout) == 1
```

Reply:
Correct now that https://github.com/labgrid-project/labgrid/pull/254 is merged